Submitted By: Robert Connolly <robert at linuxfromscratch dot org> (ashes)
Date: 2005-02-18
Initial Package Version: 3.3.4
Upstream Status: Not submitted
Origin: http://svn.exactcode.de/t2/trunk/package/base/uclibc/patches/\
	qt-uclibc.patch
Description: fenv.h does not exist in uClibc.

diff -Naur qt-x11-free-3.3.4.orig/src/tools/qlocale.cpp qt-x11-free-3.3.4/src/tools/qlocale.cpp
--- qt-x11-free-3.3.4.orig/src/tools/qlocale.cpp	2005-01-21 17:16:05.000000000 +0000
+++ qt-x11-free-3.3.4/src/tools/qlocale.cpp	2005-02-18 13:36:59.000000000 +0000
@@ -55,6 +55,10 @@
 #   undef INFINITY
 #endif
 
+#if defined(Q_OS_LINUX) && defined(__UCLIBC__)
+#   undef Q_OS_LINUX
+#endif
+
 #ifdef Q_OS_LINUX
 #    include <fenv.h>
 #endif
